test/document_stores/test_pinecone.py

```diff
@@ -1,16 +1,16 @@
 from typing import List, Union, Dict, Any
 
 import os
-from datetime import datetime
 from inspect import getmembers, isclass, isfunction
 
 import pytest
 
 from haystack.document_stores.pinecone import PineconeDocumentStore
-from haystack.schema import Document
+from haystack.schema import Document, Label, Answer
```

Comment:
I don't see where Label or Answer are used in this file. I think they should not be added here.